session7/src/com/itschool/course/Curs7Main.java

```diff
@@ -7,25 +7,17 @@
 public class Curs7Main {
 //COURSE 5-6 HOMEWORK
```

Comment:
I understand this is homework of sessions 5 and 6, but the class name at line 7 is named `Curs7Main` and it is in seesion7 module. Not a big deal, but next time please try to be consistent with these notations. 
Also, this homework is in the package `com.itschool.course`. This should be live courses reserved. For homework please create another package, something like `com.itschool.homework`.
In other words, this class would have been looked much better with this structure:
class name (for example) `Ex1`, so 
`public class Ex1`
in module `session6`, package `com.itschool.homework`

So full path: `session6/src/com.itschool.homework/Ex1.java`

Try to do it in this way next time.